lib/spandex_datadog/api_server.ex

```diff
@@ -84,7 +84,7 @@ defmodule SpandexDatadog.ApiServer do
   """
   @spec init(opts :: Keyword.t()) :: {:ok, State.t()}
   def init(opts) do
-    {:ok, agent_pid} = Agent.start_link(fn -> 0 end, name: :spandex_currently_send_count)
+    {:ok, agent_pid} = Agent.start_link(fn -> 0 end)
```

Comment:
@zachdaniel Do you know/recall whether this is actually needed for anything? I was struggling to figure out how to start up this GenServer in tests because it wants to register this Agent by name, and it can't because it's already registered in the `setup_all` block. As far as I can tell, this isn't even needed and it works just fine without it because we track the PID in the GenServer state anyway.

Reply:
Nothing comes to mind. If it works without it then I say leave it out.